src/Core/Entities/Organization.cs

```diff
@@ -258,5 +258,6 @@ public void UpdateFromLicense(OrganizationLicense license)
         UseSecretsManager = license.UseSecretsManager;
         SmSeats = license.SmSeats;
         SmServiceAccounts = license.SmServiceAccounts;
+        LimitCollectionCreationDeletion = license.LimitCollectionCreationDeletion;
```

Comment:
This just occurred to me - this needs similar feature flag logic to `OrganizationService`. The OrgService method creates an organization, this method updates an existing organization. In both cases we want to ignore the license property and use the default value unless FC has been enabled for self-hosted.